### DISCUSSION TITLE
Azuracast AutoDj - Crossfade it doesn't work properly

### DISCUSSION THREAD

**Discussion by Cris247R** [AzuraCast/AzuraCast#6578] [General]
Hello

Possible azuracast error after the last update (stable version)

After about 2 hours of perfect operation, I identified fairly large pauses between songs. So the crossfade is cancelled. There were no problems until the last update.

When I restart the radio configuration, it works perfectly for 2-3 hours.  After 2 - 3 hours again Long pauses.

My settings is:

Crossfade Method:  Normal Mode
Crossfade Duration (Seconds): 2
Server: Icecast
Server Operation system Ubuntu: 20.04

I use azuracast the latest stable version -  v0.19.1 Stable • Docker • PHP 8.2

You can check if there is a problem with this update.  Thank YOU

**Comment by BusterNeece** [comment]
Already being investigated and possibly fixed in the latest Rolling Release versions. Follow issue #6564.

**Reply by Cris247R** [reply]
Thank you
